ui/decredmaterial/votebar.go

```diff
@@ -233,12 +231,13 @@ func (v *VoteBar) LayoutWithLegend(gtx C) D {
 	)
 }
 
-func (v *VoteBar) layoutIconAndText(gtx C, lbl Label, count float32) D {
+func (v *VoteBar) layoutIconAndText(gtx C, lbl Label, count float32, clr color.NRGBA) D {
```

Comment:
```suggestion
func (v *VoteBar) layoutIconAndText(gtx C, label Label, count float32, color color.NRGBA) D {
```

I don't think it's necessary to shorten words that aren't extremely long, as it reduces code readability

There are several cases of this in the code, so i guess we can proceed with this and do a cleanup later

Reply:
This method and file has been reviewed in Pr https://github.com/planetdecred/godcr/pull/638. Also, more cleanup will be carried out when the new UI is ready.